### PR TITLE
Update N-Way Buffering Sample Readme

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Restructured the readme to match the new template. Updated markdown formatting. Moved image to “assets” folder. Updated some branding. Corrected grammar. Expanded information related to running on DevCloud. Changed sample name to match name in sample.json file. Added emulation output.

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Documentation quality

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] VSCode
- [x] DevCloud

### CODE REVIEW

**Comment by jkinsky** [comment]
Adding @jimmytwei for notices.

**Review by yuguen-intel — APPROVED** [review]

**Comment by yuguen-intel** [comment]
Waiting for @ishaghosh27's review before merging

**Review by ishaghosh27 — APPROVED** [review]

**Pull request merged**
